src/components/learner-credit-management/data/utils.js

```diff
@@ -551,3 +555,97 @@ export const isLmsBudget = (
   activeIntegrationsLength,
   isUniversalGroup,
 ) => activeIntegrationsLength > 0 && isUniversalGroup;
+
+/**
+ * Determines if the course has already started. Mostly used around text formatting for tense
+ *
+ * @param start
+ * @returns {boolean}
+ */
+export const hasCourseStarted = (start) => dayjs(start).isBefore(dayjs());
+
+/**
+ * Returns assignable course runs within the threshold of within the subsidies expiration date
+ * offset by the DAYS_UNTIL_ASSIGNMENT_ALLOCATION_EXPIRATION constant. It sorts it from the soonest expiring
+ * enroll-by date and the enroll-by date and upgrade deadline has been normalized to ISO format.
```

Comment:
Can you restate the goal differently? Based on my reading of the logic, it seems there are roughly two goals:

1. one goal related to giving accepted assignments enough time to refund before subsidy expiry, 
2. another goal related to giving the learner enough time to accept an assignment before a possibly sudden enrollment deadline, causing the learner to have less than 90 days to accept.

If the idea is that we want both of these goals met, then I think the `Math.max` needs to be `Math.min` below, right?

Reply:
Proposed logic changes would be more extensive, actually: https://github.com/openedx/frontend-app-admin-portal/pull/1292/files#r1762034441